solarforecastarbiter/datamodel.py

```diff
@@ -117,6 +117,11 @@
     if not k.startswith('VERSION IDENTIFIER') and
     not k.startswith("RESERVED")
 ]) + tuple(DERIVED_MASKS.keys())
+ALLOWED_INTERVAL_LABELS = ('beginning', 'ending', 'instant', 'event')
```

Comment:
could use `tuple(CLOSED_MAPPING.keys())`. not sure what's better. In any case I'd move them to be next to each other